test/Integration/test-js/GetProductsColumnTypesDifferentCulture/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This function verifies that serializing an item with various data types
+ * works when the language is set to a non-enUS language.
+ */
+module.exports = async function (context, req, products) {
+    context.log(JSON.stringify(products));
```

Comment:
Do we need to set the culture before calling stringify?

Reply:
Good callout - so I'm not actually aware of any way to set a culture like you can for .NET apps. JSON.stringify has its own default method of serialization which you can override though.

So instead what I'm doing is splitting this into two tests - one that just tests serialization in general with the default language and another that specifically tests with another culture set for languages that support that (i.e. just C# for now).